src/main/java/com/dakar/dakar/models/Journey.java

```diff
@@ -22,6 +25,7 @@
     @GeneratedValue(strategy = UNIQUE)
     private String id;
     @Field
+    @NotEmpty
```

Comment:
hum good to know, but i'm not sure if it's better this or the Lombok boilerplate @NonNull...